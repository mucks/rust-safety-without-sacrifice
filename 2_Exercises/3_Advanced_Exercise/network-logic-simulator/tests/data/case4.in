# Concurrent flow simulation (threads)
NODE FC weight=1 role=Observer
NODE PR weight=1 role=Observer
NODE ZU weight=1 role=Observer
LINK FC PR latency=10 capacity=100
LINK PR ZU latency=10 capacity=100
FLOW alpha from=FC to=ZU size=9 chunk=3
FLOW beta from=PR to=ZU size=6 chunk=3
FLOW gamma from=FC to=PR size=3 chunk=3
RUN_THREADS
