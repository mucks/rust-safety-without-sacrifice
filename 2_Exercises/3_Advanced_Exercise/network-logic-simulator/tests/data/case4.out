FLOW gamma: chunks=1 path=FC->PR ETA=20ms
FLOW beta: chunks=2 path=PR->ZU ETA=20ms
FLOW alpha: chunks=3 path=FC->PR->ZU ETA=40ms
COMPLETE (threads): gamma, beta, alpha
